src/service/token.service.js

```diff
@@ -0,0 +1,29 @@
+import { Token } from '../modules/Token.js';
+
+async function save(userId, newToken) {
```

Comment:
Consider adding error handling for database operations to handle potential issues such as connection errors or validation failures.